### ISSUE TITLE
Culture is (suddenly) not supported

### ISSUE THREAD

**Issue by yeykawb** [TabularEditor/TabularEditor3#576]
Hey,

was hoping someone could shed some light on this error message.

![image](https://user-images.githubusercontent.com/97282358/175013304-c5812618-85c3-4798-b5ed-b8cf74674699.png)

The only difference is new tables and relationships between them.
The culture setting has never been changed, and is set to EN-US.

/ Jakob

**Comment by yeykawb** [comment]
Update. This issue arises when creating a specific relationship between two tables. Anyone got a clue as to why?

**Comment by albertospelta** [comment]
@yeykawb This problem probably does not depend on the culture of the model. 
What is your Windows UI language/culture ?

@otykier FYI We encountered the same issue with Bravo and this is the PR that fixed it https://github.com/sql-bi/Bravo/pull/353

**Comment by yeykawb** [comment]
Windows is using English also.
Again, this is only happening when creating a relationship between table A and B.
But, I've realized that the relationship is unnecessary, removing this removes the error.
However, it would be interesting to understand why it arises.

**Comment by yeykawb** [comment]
Ok, so the issue came back, and I'm having a hard time tracing it to something in particular since the error message keeps stating "Culture" as the issue.

@otykier, if you have any idea on what this could be caused of, I'd be highly appreciated.

**Comment by otykier** [comment]
@yeykawb which version of TE3 are you using? Any chance you could share a .bim file with support@tabulareditor.com with step-by-step instructions on how to trigger the error?

**Comment by yeykawb** [comment]
I'm using 3.3.0.
I can look into that, yes.

The issue arises when trying to create a 1:M relationship between the Session table and the 3 fact tables.
![image](https://user-images.githubusercontent.com/97282358/175059899-acd23681-beb6-4729-a9d9-03afadd1c3f8.png)

Without relationship = Saving works
With relationship = Culture is wrong

Any wild guess?

**Comment by otykier** [comment]
I don’t think it’s related to any specific relationship or other data model feature. My concern is that it’s an issue in the .NET Core version of the AMO/TOM libraries. Could you check if you can provoke the same issue in TE2 or an earlier version of TE3? If not, I think it is indeed a .NET Core issue in which case I’ll have to reach out to Microsoft to figure out what we can do.

**Comment by yeykawb** [comment]
Ok, I tested on an earlier version of TE3.
The error message made much more sense now.
![image](https://user-images.githubusercontent.com/97282358/175066104-a2b72086-6c40-4ac0-8d60-8e70d9f41ce0.png)

It was basically a blunder by me, the view contained a lot of nulls on the one side. 
Thanks for your input on trying an earlier version!

/ Jakob

**Comment by otykier** [comment]
Right, thanks for letting us know. I’ll look into why we get a different error message in 3.3.0.

**Comment by otykier** [comment]
Just an update on this. If you get the **Culture is not supported** error, add the following option under `configProperties` in the `TabularEditor3.runtimeconfig.json` file, in the installation folder:

```json
"System.Globalization.UseNls": true
```

The final file should look something like this:

```json
{
  "runtimeOptions": {
    "tfm": "net6.0",
    "frameworks": [
      {
        "name": "Microsoft.NETCore.App",
        "version": "6.0.0"
      },
      {
        "name": "Microsoft.WindowsDesktop.App",
        "version": "6.0.0"
      }
    ],
    "configProperties": {
      "System.Reflection.Metadata.MetadataUpdater.IsSupported": false,
      "System.Globalization.UseNls": true
    }
  }
}
```

**Comment by otykier** [comment]
This is fixed in [3.3.4](https://docs.tabulareditor.com/te3/other/release-notes/3_3_4.html), thanks for reporting!
